R-package/tests/testthat/test_Predictor.R

```diff
@@ -114,6 +114,166 @@ test_that("start_iteration works correctly", {
     expect_equal(pred_leaf1, pred_leaf2)
 })
 
+test_that("predict() params should override keyword argument for raw-score predictions", {
+  data(agaricus.train, package = "lightgbm")
+  X <- agaricus.train$data
+  y <- agaricus.train$label
+  bst <- lgb.train(
+    data = lgb.Dataset(
+      data = X
+      , label = y
+      , params = list(
+        verbosity = VERBOSITY
```

Comment:
I think `VERBOSITY` from `train()` function is passed to unconstructed `Dataset` class automatically, no need to specify this param twice

https://github.com/microsoft/LightGBM/blob/b462d0a40f6978c0a66a47af4522c2ae12532316/R-package/R/lgb.train.R#L80-L84
https://github.com/microsoft/LightGBM/blob/b462d0a40f6978c0a66a47af4522c2ae12532316/R-package/R/lgb.train.R#L152-L154

Reply:
ah good point! updated in https://github.com/microsoft/LightGBM/pull/5122/commits/d8efeaab40d5bcfa1b082304ca47f0bab20005f5